package.json

```diff
@@ -52,7 +54,7 @@
     "chai": "^4.1.2",
     "gulp": "^4.0.0",
     "mocha": "^5.0.0",
-    "typescript": "^3.0.3"
```

Comment:
I suppose leaving them in devDependencies would be another option.

Reply:
Yes I prefer that option, it IS a dev dependency, we need it to run tests. In "production", it's up to the user to install the plugins he needs.